dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -643,16 +615,10 @@ def expect_controller_logs_milestone_regexp(regexp)
       end
     end
 
-    # record activity in session
-    assert_equal 100, client_state.level_progress(@script_level.level_id)
-
-    # record the total lines of code in session
-    assert_equal 20, client_state.lines
-
     assert_response :success
 
     expected_response = build_expected_response(
-        total_lines: 20,
+        total_lines: 0,
```

Comment:
Is it the case that the response includes total_lines but it is no longer meaningful since it is computed client side? 
Might be better to just eliminate that from the response and make sure there is no client code referencing it, since if there were such client code, that code would be referencing bogus values and buggy.